j5/vision/markers.py

```diff
@@ -0,0 +1,203 @@
+"""Markers Class."""
+
+from __future__ import annotations
+
+from typing import List, NamedTuple, Union
+
+import numpy as np
+
+
+class Cartesian(NamedTuple):
+    """A Cartesian Coordinate."""
+
+    x: float
+    y: float
+    z: float
+
+    def as_cartesian(self) -> Cartesian:
+        """Provides the coordiante in the form of a cartesian coordinate."""
+        return self
```

Comment:
I added this for for add_markers method so that it was coordinate system agnostic. I would also like to be able to return coordinates using any system, so that teams can use the coordinate system that best suits them and the control logic.

I plan to the remove the markers class' static conversion methods, because these are made redundant by the casting interface and I don't think the convertions should be handled by the markers class.